src/main/java/org/tron/p2p/connection/business/detect/NodeDetectService.java

```diff
@@ -13,11 +23,6 @@
 import org.tron.p2p.discover.Node;
 import org.tron.p2p.discover.NodeManager;
 
-import java.net.InetAddress;
-import java.net.InetSocketAddress;
-import java.util.*;
-import java.util.concurrent.*;
-
 @Slf4j(topic = "net")
```

Comment:
It is advised that a new PR

Reply:
why? Only format the code.

Reply:
`don't submit new task if server is closing`: It doesn't seem to have anything to do with this.